src/Cake.Common/Tools/NuGet/Pack/NuGetPacker.cs

```diff
@@ -214,6 +214,12 @@ private ProcessArgumentBuilder GetArguments(FilePath filePath, NuGetPackSettings
                     settings.Properties.Select(property => string.Concat(property.Key, "=", property.Value))));
             }
 
+            // Keep temporarily autogenerated NuSpec file?
+            if (settings.KeepTemporaryNuSpecFile)
+            {
+                builder.Append("-KeepTemporaryNuSpecFile");
```

Comment:
Good question. I meant to add the parsing logic for it as well, but forgot since I didn't immediately figure out how it was done for `RequireLicenseAcceptance`. Pointers to how and where the argument parsing is performed?